shapely/tests/test_strtree.py

```diff
@@ -964,468 +1234,124 @@ def test_query_contains_properly_points(tree, geometry, expected):
     assert_array_equal(
-        line_tree.query_items(geometry, predicate="contains_properly"), expected
+        line_tree.query(geometry, predicate="contains_properly"), expected
     )
 
 
 @pytest.mark.parametrize(
     "geometry,expected",
     [
         # point does not contain any polygons (not valid relation)
-        (shapely.points(0, 0), []),
+        (Point(0, 0), []),
+        ([Point(0, 0)], [[], []]),
         # line intersects multiple polygons but does not contain any (not valid relation)
         (shapely.linestrings([[0, 0], [2, 2]]), []),
+        ([shapely.linestrings([[0, 0], [2, 2]])], [[], []]),
         # box overlaps envelope of 2 polygons but contains neither
         (box(0, 0, 1, 1), []),
+        ([box(0, 0, 1, 1)], [[], []]),
         # larger box intersects 3 polygons but contains only one
         (box(0, 0, 2, 2), [1]),
+        ([box(0, 0, 2, 2)], [[0], [1]]),
         # buffer overlaps 3 polygons but contains none
-        (shapely.buffer(shapely.points(3, 3), HALF_UNIT_DIAG), []),
+        (shapely.buffer(Point(3, 3), HALF_UNIT_DIAG), []),
+        ([shapely.buffer(Point(3, 3), HALF_UNIT_DIAG)], [[], []]),
         # larger buffer overlaps 6 polygons (touches midpoints) but contains one
-        (shapely.buffer(shapely.points(3, 3), 3 * HALF_UNIT_DIAG), [3]),
+        (shapely.buffer(Point(3, 3), 3 * HALF_UNIT_DIAG), [3]),
+        ([shapely.buffer(Point(3, 3), 3 * HALF_UNIT_DIAG)], [[0], [3]]),
         # envelope of points overlaps polygons, but points do not intersect
         # (not valid relation)
-        (shapely.multipoints([[5, 7], [7, 5]]), []),
+        (MultiPoint([[5, 7], [7, 5]]), []),
+        ([MultiPoint([[5, 7], [7, 5]])], [[], []]),
     ],
 )
 def test_query_contains_properly_polygons(poly_tree, geometry, expected):
     assert_array_equal(
-        poly_tree.query_items(geometry, predicate="contains_properly"), expected
+        poly_tree.query(geometry, predicate="contains_properly"), expected
     )
 
 
 ### predicate = 'dwithin'
 
 
-@pytest.mark.skipif(shapely.geos_version >= (3, 10, 0), reason="GEOS >= 3.10")
-def test_query_dwithin_geos_version(tree):
-    with pytest.raises(UnsupportedGEOSVersionError, match="requires GEOS >= 3.10"):
-        tree.query_items(shapely.points(0, 0), predicate="dwithin", distance=1)
-
-
-@pytest.mark.skipif(shapely.geos_version < (3, 10, 0), reason="GEOS < 3.10")
-@pytest.mark.parametrize(
-    "distance,match",
-    [
-        (None, "distance parameter must be provided"),
-        ("foo", "could not convert string to float"),
-        ([1.0], "distance must be a scalar value"),
-        ([None], "distance must be a scalar value"),
-    ],
-)
-def test_query_dwithin_invalid_distance(tree, distance, match):
-    with pytest.raises(ValueError, match=match):
-        tree.query_items(shapely.points(0, 0), predicate="dwithin", distance=distance)
-
-
-@pytest.mark.skipif(shapely.geos_version < (3, 10, 0), reason="GEOS < 3.10")
```

Comment:
These tests are mostly intact, but moved around just a little bit as part of merging them; unfortunate that these are showing up as being deleted rather than updated.  These are showing up a bit further down on the right in GH review panel (e.g., line 1330)